### PR TITLE
Implement sequence player

### PR DESCRIPTION
**Pull request by namse**
# What I did
I ported sequence player from ts to rust. very similar, nothing very new.

### CODE REVIEW

File: luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/image_browser/mod.rs [hunk 1]
```diff
@@ -53,44 +53,36 @@ impl ImageBrowser {
     pub fn update(&mut self, event: &dyn std::any::Any) {
         if let Some(event) = event.downcast_ref::<EditorEvent>() {
             match event {
```
Comment (namse) [inline luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/image_browser/mod.rs:55]: Dont spend your time to check this file, I just remove log and rustfmt update the format.

File: luda-editor/luda-editor-client/Cargo.toml [hunk 2]
```diff
@@ -25,6 +25,7 @@ luda-editor-rpc = { path = "../luda-editor-rpc" }
 num = "0.4.0"
 once_cell = "1.8.0"
 auto_ops = "0.3"
+ordered-float = "2.10"
```
Comment (namse) [inline luda-editor/luda-editor-client/Cargo.toml:28]: For floating number eq, ord

**Comment by namse** [comment]
I will open this PR after merging https://github.com/NamseEnt/namseent/pull/89.

File: luda-editor/luda-editor-client/src/app/editor/sequence_player/content_loader.rs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                                loading_contents.push_back(LoadingContent::Image(url));
+                            }
+                            namui::ImageSource::Image(_) => {
+                                continue;
+                            }
+                        }
+                    }
+                }
+                Track::Subtitle(_) => {
+                    // NOTE: namui starts engine after loading fonts.
```
Comment (namse) [inline luda-editor/luda-editor-client/src/app/editor/sequence_player/content_loader.rs:54]: This mean, nothing to load for subtitle.

File: luda-editor/luda-editor-client/src/app/editor/sequence_player/mod.rs [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+            style: namui::RectStyle {
+                stroke: Some(namui::RectStroke {
+                    color: Color::BLACK,
+                    border_position: namui::BorderPosition::Inside,
+                    width: 1.0,
+                }),
+                ..Default::default()
+            },
+        });
+        let title_header = namui::text(namui::TextParam {
```
Comment (namse) [inline luda-editor/luda-editor-client/src/app/editor/sequence_player/mod.rs:167]: I think we don't need title header. do you think so?

Comment (bigfoodK) [reply luda-editor/luda-editor-client/src/app/editor/sequence_player/mod.rs:167]: I thnk so.
In near future, sequence title will be displayed on top bar. maybe...

File: luda-editor/resources/sequence/testSequence.json [hunk 5]
```diff
@@ -198,7 +198,7 @@
                     {
                         "id": "s-2-clip",
                         "start_at": {
-                            "milliseconds": 0
+                            "milliseconds": 1000
```
Comment (bigfoodK) [inline luda-editor/resources/sequence/testSequence.json:201]: Is it intended to include this change?

File: luda-editor/luda-editor-client/src/app/editor/mod.rs [hunk 6]
```diff
@@ -289,14 +314,19 @@ impl Editor {
...
+        let sequence_rc = Rc::new(sequence.clone());
         Self {
             timeline: Timeline::new(),
-            playback_time: chrono::Duration::zero(),
             image_filename_objects: vec![],
             job: None,
             clip_editor: None,
             selected_clip_id: None,
-            sequence,
+            sequence, // NOTE : I think editor should not have mutable sequence, but just immutable to track changes
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/editor/mod.rs:324]: It's weird...
mutable sequence, but just immutable to track?
Isn't the track in the sequence?
How is this possible?

**Review by bigfoodK — CHANGES_REQUESTED** [review]
image
Let's remove this huuuuuge title

File: luda-editor/resources/sequence/testSequence.json (hunk 5 above)
Comment (namse) [reply luda-editor/resources/sequence/testSequence.json:201]: Yes, is it. not important, but for test.

File: luda-editor/luda-editor-client/src/app/editor/mod.rs (hunk 6 above)
Comment (namse) [reply luda-editor/luda-editor-client/src/app/editor/mod.rs:324]: track, not noun, for verb. -> for tracking.
I mean, sequence should be immutable so we can take a history of sequence's changes, so can implement undo-redo system (by tracking changes).

**Comment by namse** [comment]
@bigfoodK I removed title. I kept buttons. If you want to remove buttons for sequence list preview, I suggest to make a boolean field in SeuqncePlayerProps and remove it in render funciton.

**Review by bigfoodK — APPROVED** [review]
Good

**Pull request merged**